lib/assessment.js

```diff
@@ -8,15 +8,19 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(array, callback) {
+  for (var i = 0, i <numbers.length; i++) {
+    if(numbers [i]  < min) {
+      min = numbers[i];
+    }
+  }
```

Comment:
This code will give you the minimum value in an array. However, the question was specifically asking you to write a **callback** that could be passed into `.reduce`. 
e.g.
```javascript
var min = function(p, c) {
 ...
};
[1,2,3].reduce(min);
```